scipy/special/sf_error.pxd

```diff
@@ -12,6 +12,8 @@ cdef extern from "sf_error.h":
         DOMAIN "SF_ERROR_DOMAIN"
         ARG "SF_ERROR_ARG"
         OTHER "SF_ERROR_OTHER"
+        NOMEM "SF_ERROR_NOMEM"
```

Comment:
Bikeshed, but I think this would read better as `NO_MEM`. I think `SF_ERROR_MEMORY` fits better with the other names though.

Reply:
That sounds good to me. "NOMEM" is a holdover from Posix [`ENOMEM`](https://en.wikipedia.org/wiki/Errno.h#POSIX_errors), but `SF_ERROR_MEMORY` is good too. Any preference on the string that would be used in `geterr()` and `seterr()` (which is public Python API).  I used `"nomem"`, but `"memory"` or `"no_memory"` are also reasonable.

Reply:
I like `"memory"`. It seems consistent with `MemoryError` for Python's relevant exception class. I wasn't aware of the Posix symbolic error names but figured  `"nomem"` probably came from somewhere. It was kind of funny reading through the list wondering if the names had a character limit and then finding `ENAMETOOLONG`.